docs/userGuide/syntax/mathformulae.mbdf

```diff
@@ -0,0 +1,99 @@
+## Math Formulae
+Markbind supports typesetting TeX math equations. [KaTeX](https://katex.org) is used as a fast math renderer.
+
+<box type="info">
+
+Insert `inline` equations by enclosing them in round brackets `\( ... \)`.
+
+Insert `display` equations by enclosing them in square brackets `\[ ... \]`.
+
+Insert numbered `display` equations by enclosing the equation square brackets and the equation number in curly brackets `\[ ... \] (1)`.
+</box>
+
+<box type="important">
+
+When typing `display` equations, this does not work as well:
+
+```markdown
+abc
+\[2+2=4\]
+```
+
+or this
+```markdown
+> abc
+> \[2+2=4\]
+```
+
+If your equation requires special Nunjucks tags like {% raw %}`{{`{% endraw %} or {% raw %}`}}`{% endraw %},use a
+[raw-endraw block together with v-pre](https://markbind.org/userGuide/tipsAndTricks.html#using-raw-endraw-to-display-content):
+
+```markdown
+{% raw %}\(e^{{\frac{1}{3}} + 1}\){% endraw %} {v-pre}
+```
+
+</box>
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">markdown</variable>
+<variable name="code">
+\( e^{i\pi}+1=0 \)
+\(\frac{1}{3}+\frac{2}{3}=1\)
+
+\[ e^{i\pi}+1=0 \]
+\[\frac{1}{3}+\frac{2}{3}=1\] (1)
+
+{% raw %}\(e^{{\frac{1}{3}} + 1}\){% endraw %} {v-pre}
+
+Bad examples:
+
+\(1+1=2\)
+\[2+2=4\]
+
+> abc
+> \[2+2=4\]
+
+</variable>
+</include>
+
+<small>More info on allowed symbols: https://katex.org/docs/support_table.html</small>
+
+<span id="short" class="d-none">
+
+```markdown
+\( e^{i\pi}+1=0 \)
+\(\frac{1}{3}+\frac{2}{3}=1\)
+
+\[ e^{i\pi}+1=0 \]
+\[\frac{1}{3}+\frac{2}{3}=1\] (1)
+
+{% raw %}\(e^{{\frac{1}{3}} + 1}\){% endraw %} {v-pre}
+
+Bad examples:
+
+\(1+1=2\)
+\[2+2=4\]
+
+> abc
+> \[2+2=4\]
+
+```
+</span>
+<span id="examples" class="d-none">
+
+\( e^{i\pi}+1=0 \)
+\(\frac{1}{3}+\frac{2}{3}=1\)
+
+\[ e^{i\pi}+1=0 \]
+\[\frac{1}{3}+\frac{2}{3}=1\] (1)
+
+{% raw %}\(e^{{\frac{1}{3}} + 1}\){% endraw %} {v-pre}
+
+Bad examples:
```

Comment:
let's remove this too,
instead make the examples directly above here resemble actual use cases as best; i.e. some _short_ text around an inline equation, then present another relavant equation that is better highlighted in display format